src/quacc/runners/ase.py

```diff
@@ -36,7 +36,11 @@ class VibKwargs(TypedDict, total=False):
 
 
 def run_calc(
-    atoms: Atoms, geom_file: str | None = None, copy_files: list[str] | None = None
+    atoms: Atoms,
+    geom_file: str | None = None,
+    copy_files: list[str] | None = None,
+    tmpdir: str | None = None,
+    calc_clean: bool = True,
```

Comment:
See comments below about the file I/O stuff. I'm not yet convinced that this is ideal.